Source/hu_stuff.c

```diff
@@ -1700,6 +1703,11 @@ void HU_Ticker(void)
   if ((showMessages || message_dontfuckwithme) && plr->message &&
       (!message_nottobefuckedwith || message_dontfuckwithme))
     {
+      if (message_centered)
```

Comment:
If you hit Enter to display a previous message, will this also appear properly centered?

Reply:
Yes, it works.